app/src/main/java/com/geeksville/mesh/ui/components/NodeMenu.kt

```diff
@@ -0,0 +1,118 @@
+package com.geeksville.mesh.ui.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Row
+import androidx.compose.material.Checkbox
+import androidx.compose.material.CheckboxDefaults
+import androidx.compose.material.Divider
+import androidx.compose.material.DropdownMenu
+import androidx.compose.material.DropdownMenuItem
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import com.geeksville.mesh.R
+import com.geeksville.mesh.database.entity.NodeEntity
+
+@Suppress("LongMethod")
+@Composable
+fun NodeMenu(
+    node: NodeEntity,
+    ignoreIncomingList: List<Int>,
+    isThisNode: Boolean = false,
+    onMenuItemAction: (MenuItemAction) -> Unit,
+    onDismissRequest: () -> Unit,
+    expanded: Boolean = false,
+) {
+    val isIgnored = ignoreIncomingList.contains(node.num)
+
+    DropdownMenu(
+        modifier = Modifier.Companion.background(MaterialTheme.colors.background.copy(alpha = 1f)),
```

Comment:
Companion?

Reply:
Gemini likes to add that in autocomplete, for some reason.. missed this one